prestoadmin/collect.py

```diff
@@ -209,10 +210,10 @@ def system_info():
 
 def get_system_info(download_location):
 
-    if not exists(TMP_PRESTO_DEBUG):
-        run("mkdir " + TMP_PRESTO_DEBUG)
+    if not exists(TMP_PRESTO_DEBUG_REMOTE):
```

Comment:
it actually is related -- on the master, /tmp/presto-debug has already been created and is owned by root. when presto-admin logs into all the nodes and tries to create /tmp/presto-debug/sysinfo.txt with a non-root user, it fails